### ISSUE TITLE
Client crashes when clicking on Radiative Ingot in NEI 

### ISSUE THREAD

**Issue by notchriss** [ReikaKalseki/Reika_Mods_Issues#1647]
Clicking on any other Chromaticraft Alloy works fine and displays recipe as expected. Also when I click on any other alloy and use Chromaticraft interface arrows to scroll to Radiative Ingot, everything works fine too, no crashes. 

https://pastebin.com/LCuG72TF

**Comment by ReikaKalseki** [comment]
I have seen this too, but as a result of having added custom alloying recipes. Do you have custom recipes?

**Comment by notchriss** [comment]
I dont think so. I've put this pack together from scratch myself and the only recipe I've ever changed was diamond chip from OpenComputers due to recipe conflict with EnderIo. Nobody besides me has access to the server. Unless some other mod changed it automatically but I doubt it very much.

**Comment by ReikaKalseki** [comment]
There is no way the recipe lists should be of different sizes between client and server...

**Comment by notchriss** [comment]
I have a strong suspicion it was MineTweaker's doing, even if I didn't use it. After I removed MineTweaker while trying to debug another issue, Radiative Ingot in NEI works fine and so does Chromastone (it used to crash too I just learned it after I originally posted the issue). I also removed some other mods but none of them seem to be remotely related to recipes. Whenever I have time and empty server I may try re adding it to see if it causes crashes again.

**Comment by ReikaKalseki** [comment]
This is an internal recipe list and handler that MT does not know about...

**Comment by notchriss** [comment]
In that case I have no other ideas...
